docs/source/docker_only.rst

```diff
@@ -0,0 +1,129 @@
+
+Start adminer:
+
+.. code-block:: shell
+
+    docker run -d -p 8080:8080 --name dallinger_adminer --network dallinger --link dallinger_postgres:db adminer
+
+Connect to adminer by visiting http://localhost:8080
+Select PostgreSQL from the dropdown, and enter `dallinger` as both username and password. You can leave the "database" field blank to get a list of existing databases.
+
+
+Run experiment from docker image
+================================
+
+Enter the experiment directory:
+
+.. code-block:: shell
+
+   cd /path/to/experiment-directory
+
+Create a file named `Dockerfile` with these contents (replace image name in the `FROM` directive to match the dallinger version you depend on in your `constraints.txt` file):
+
+.. code-block::
+
+    # syntax = docker/dockerfile:1.2
+    FROM ghcr.io/dallinger/dallinger:9.1.0a1
+
+    RUN mkdir /experiment
+    COPY requirements.txt /experiment/requirements.txt
+    COPY constraints.txt /experiment/constraints.txt
+
+    WORKDIR /experiment
+
+    RUN python3 -m pip install -e /dallinger[docker]
+
+    # Some experiments might only list dallinger as dependency
+    # If they do the grep command will exit non-0, the pip command will not run
+    # but the whole `RUN` group will succeed thanks to the last `true` invocation
+    RUN grep -v ^dallinger requirements.txt > /tmp/requirements_no_dallinger.txt && \
+        python3 -m pip install -r /tmp/requirements_no_dallinger.txt -c constraints.txt || true
+
+    COPY . /experiment
+    ENV PORT=5000
+
+    CMD dallinger_heroku_web
+
+Build a docker image for the experiment using Buildkit:
+
+.. code-block:: shell
+    EXPERIMENT_IMAGE=my-experiment
+    DOCKER_BUILDKIT=1 docker build . -t ${EXPERIMENT_IMAGE}
+
+Start the development server with docker:
+
+.. code-block:: shell
+
+    docker run --name dallinger --rm -ti -u $(id -u ${USER}):$(id -g ${USER}) -v ${PWD}:/experiment --network dallinger -p 5000:5000 -e FLASK_OPTIONS='-h 0.0.0.0' -e REDIS_URL=redis://dallinger_redis:6379 -e DATABASE_URL=postgresql://dallinger:dallinger@dallinger_postgres/dallinger ${EXPERIMENT_IMAGE} dallinger develop debug
+
+You can now access the running dallinger instance on http://localhost:5000/dashboard
+The admin password can be found in the develop `config.txt` file:
+
+.. code-block:: shell
+
+    grep dashboard_password ./develop/config.txt
+
+
+Deploy the experiment image using ssh
+=====================================
+
+We're going to use variations of the same command, so we create an alias for convenience.
+
+.. code-block:: shell
+    # On Linux you can use:
+    alias docker-dallinger='docker run --rm -ti -v /etc/group:/etc/group -v ~/.docker:/root/.docker -v ~/.local/share/dallinger/:/root/.local/share/dallinger/ -e HOME=/root -e DALLINGER_NO_EGG_BUILD=1 -v /var/run/docker.sock:/var/run/docker.sock -v $(readlink -f $SSH_AUTH_SOCK):/ssh-agent -e SSH_AUTH_SOCK=/ssh-agent -v ${PWD}:/experiment  ${EXPERIMENT_IMAGE} dallinger'
```

Comment:
@silviot I cannot get this to work on Ubuntu 22.04. I changed the command slightly like proposed in this post:
https://gist.github.com/d11wtq/8699521?permalink_comment_id=4090122#gistcomment-4090122
It returns
`/bin/bash: /home/frank/projects/PsyNet/demos/mcmcp:/experiment: No such file or directory`
though.